ui/DesignSystem/Component/Transaction.svelte

```diff
@@ -8,12 +8,15 @@
   //   stake: "name of stake (optional)",
   //   subject: {
   //     name: "name of the transaction target",
-  //     kind: "project" || "user"
-  //     avatar: "avatar url of the target (optional)"
+  //     kind: "project" || "user",
+  //     avatarFallback: "avatar fallback of the target",
+  //     imageUrl: "avatar url of the target (optional)"
   //   },
   //   payer: {
   //     name: "name of the owner of the paying wallet"
-  //     avatar: "avatar of the owner (optional)"
+  //     avatar: "avatar of the owner (optional)",
+  //     avatarFallback: "avatar fallback of the owner",
+  //     imageUrl: "avatar url of the owner (optional)"
   //   }
   // }
```

Comment:
Is this comment block a leftover or should it stay here for documentation purposes?

Reply:
For documentation purposes. It should be possible to look up the variations of the content of `tx` without having to go through the whole code.